### ISSUE TITLE
Operator docs point to development builds

### ISSUE THREAD

**Issue by howardjohn** [istio/istio.io#5532]
https://preliminary.istio.io/docs/setup/install/standalone-operator/ points to https://preliminary.istio.io/operator.yaml which points to 1.4-dev images.

This needs to be changed to the release binary

**Comment by ostromart** [comment]
I don't think we have any way of doing this apart from manual changes atm. Any objection to keeping this pointing to 1.4-dev until after we have our "final" RC?

**Comment by howardjohn** [comment]
Are we going to continue to update this every single release for the next 6-9 months for every patch release? If we cannot ensure we will do that then we should point to the release binaries instead. This is experimental though, so not too bad if its pointing to the wrong thing

I think the end plan is to have this as part of istioctl anyways though, right?

**Comment by ostromart** [comment]
No, we definitely need a plan, I'm just proposing this until we have a solution. 
Yes, we will have istioctl operator init and if we are fine with that being the only way to install then we are ok.

**Comment by davidebbo** [comment]
One workaround is to use `hub` and `tag` to get the official 1.4.0 build instead of the dev build. e.g.

```yaml
apiVersion: install.istio.io/v1alpha2
kind: IstioControlPlane
metadata:
  namespace: istio-operator
  name: example-istiocontrolplane
spec:
  profile: default
  hub: docker.io/istio
  tag: 1.4.0
```

**Comment by dgn** [comment]
As we're not going to fix this for 1.5 I'm moving to 1.6

**Comment by howardjohn** [comment]
This is fixed
